bedrock/mozorg/templates/mozorg/home/home-contentful.html

```diff
@@ -1,142 +0,0 @@
-{#
- This Source Code Form is subject to the terms of the Mozilla Public
- License, v. 2.0. If a copy of the MPL was not distributed with this
- file, You can obtain one at https://mozilla.org/MPL/2.0/.
-#}
-
-{% from "macros-protocol.html" import hero, call_out_compact, card with context %}
-
-{% from "mozorg/home/includes/macros.html" import download_banner, download_banner_secondary, fxa_banner %}
-
-{% extends "mozorg/home/base.html" %}
-
-{# Bug 1438302, Issue 13019: Avoid duplicate content for English pages. #}
-{%- block page_title_full -%}
-  {%- if LANG == 'en-US' -%}
-    Internet for people, not profit — Mozilla (US)
-  {%- elif LANG == 'en-GB' -%}
-    Internet for people, not profit — Mozilla (UK)
-  {%- else -%}
-    Internet for people, not profit - Mozilla
-  {%- endif -%}
-{%- endblock -%}
-{% block page_title_suffix %}{% endblock %}
-
-{% block page_desc %}Mozilla is the not-for-profit behind the lightning fast Firefox browser. We put people over profit to give everyone more power online.{% endblock %}
-
-{% set show_fundraising_banner = switch('fundraising-banner-giving2022') and ftl_has_messages('banner-fundraising-title', 'banner-fundraising-body') %}
-{% set show_firefox_app_store_banner = switch('firefox-app-store-banner') %}
-
-{% block page_css %}
-  {% include 'includes/contentful/css.html' %}
-  {{ css_bundle('protocol-call-out') }}
-  {{ css_bundle('protocol-newsletter') }}
-  {{ css_bundle('home-contentful') }}
-
-  {% if show_fundraising_banner %}
-    {{ css_bundle('fundraising-banner') }}
-  {% elif show_firefox_app_store_banner %}
-    {{ css_bundle('firefox-app-store-banner') }}
-  {% endif %}
-
-  {% if switch('homepage-pocket-promo', ['en-US']) %}
-    {{ css_bundle('home-pocket-promo') }}
```

Comment:
Can this `home-pocket-promo` CSS bundle be removed too?

Reply:
Spoke to Dan and he did mention to keep these promo files (Pocket promo and CTD promo) in the codebase in case they do need to come up again soon after launching the new homepage. In the meantime, we're not going to be using these promotions. We can remove them from our codebase a month or so after launch, if we don't face any hiccups.